src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -1781,8 +1811,14 @@ export function getChartOptions(
         };
     }
 
-    const tooltipFn = isTreemap(type) ? generateTooltipTreemapFn(viewByAttribute, stackByAttribute, config) :
-        generateTooltipFn(viewByAttribute, type, config);
+    let tooltipFn;
+    if (isTreemap(type)) {
+        tooltipFn = generateTooltipTreemapFn(viewByAttribute, stackByAttribute, config);
+    } else if (isViewByTwoAttributes) {
+        tooltipFn = generateTooltipForTwoAttributesFn(viewByAttribute, viewByParentAttribute, config);
```

Comment:
This change is not unit tested. You test the function itself correctly but you need to add a test to changed behavior of the `getChartOptions` as well (that your new tooltip function is assigned to the charts options object in a case when optional stacking chart is being built).

Reply:
The new unit test was added here https://github.com/gooddata/gooddata-react-components/pull/881/commits/958a8d8e97d46b9027c5d60495423825fd3175d2